### PR TITLE
Brazilian Portuguese Language

### PR DESCRIPTION
**Pull request by caiodias**
I reviewed all strings to language br (Brazilian Portuguese language)

### CODE REVIEW

File: languages.lua [hunk 1]
```diff
@@ -1283,179 +1283,179 @@ return {
...
-            member = '&&&1 é um membro do grupo'
+            member = '&&&1 é um membro do grupo.'
         },
         getban = {
-            header = '*Global stats* for ',
-            nothing = '`Nothing to display`',
-            kick = 'Kick: ',
-            ban = 'Ban: ',
-            tempban = 'Ban temporàrio: ',
+            header = '*Status global * para ',
```
Comment (TiagoDanin) [inline languages.lua]: - [Note]Two spaces?

File: languages.lua [hunk 2]
```diff
@@ -1464,8 +1464,8 @@ return {
             modlist = '*Criador*:\n&&&1\n\n*Admins*:\n&&&2'
         },
         report = {
-            no_input = 'Envie suas sugestões/bugs/dúvidas com !\nExample: !hello, this is a test',
-            sent = '*Feedback enviado!*',
+            no_input = 'Envie suas sugestões/bugs/dúvidas com !\nExamplo: !olá, este é um teste',
```
Comment (TiagoDanin) [inline languages.lua]: - [Note] 'Exemplo' is the correct

File: languages.lua [hunk 3]
```diff
@@ -1536,9 +1536,9 @@ return {
                 ram = 'Nova configuração para a mensagem de boas-vindas:\n*Regras*\n*Descrição*\n*Lista de moderadores(as)*',
                 no = 'Nova configuração para a mensagem de boas-vindas:\nRegras\nDescrição\nLista de moderadores(as)',
                 wrong_input = 'Argumento inválido.\nUse _/welcome [no|r|a|ra|ar]_',
-                custom = '*Custom welcome message* setted!\n\n&&&1',
-                custom_setted = '*Custom welcome message saved!*',
-                wrong_markdown = '_Not setted_ : I can\'t send you back this message, probably the markdown is *wrong*.\nPlease check the text sent',
+                custom = '*Mensagem de boas-vindas personalizada* configurada!\n\n&&&1',
+                custom_setted = '*ensagem de boas-vindas personalizada salva!*',
```
Comment (TiagoDanin) [inline languages.lua]: - [Note] Missing the M in `'*ensagem de` to `*Mesagem de`
- Same on lina 1540

File: languages.lua [hunk 4]
```diff
@@ -1631,34 +1631,34 @@ return {
...
+            sent = '_Enviei para você o menu anti-flood no privado_',
+            ignored = '[&&&1] será ignorado pelo sistema anti-flood',
+            not_ignored = '[&&&1] não será ignorado pelo sistema anti-flood',
+            number_cb = 'Sensibilidade atual. Toque em cima do + ou do -',
+            header = 'Você pode gerenciar as configurações de flood do grupo daqui.\n'
+                ..'\n*primeira linha*\n'
+                ..'• *ON/OFF*: o atual status do sistema anti-flood\n'
+                ..'• *Kick/Ban*: O que será feito quando alguém estiver floodando\n'
+                ..'\n*segunda linha*\n'
+                ..'• Voc6e pode usar *+/-* para mudar a sensibilidade atual do sistema anti-flood\n'
```
Comment (TiagoDanin) [inline languages.lua]: - [Note] `'• Voc6e pode usar *+/` The correct is "Você"

**Comment by TiagoDanin** [comment]
I did some notes in [Files changed](https://github.com/RememberTheAir/GroupButler/pull/33/files) :D

**Comment by caiodias** [comment]
Thank you so much @TiagoDanin I will make the changes and commit again.

Please @RememberTheAir wait until my next commit.

**Comment by RememberTheAir** [comment]
Okay
Thanks a lot for the translation guys :)

**Comment by caiodias** [comment]
Thank you again @TiagoDanin.

@RememberTheAir I already made the changes, the Pull Request is ready to be merged when you want.

**Pull request merged**